### ISSUE TITLE
error in Samsung Galaxy tab A 8.0 2017

### ISSUE THREAD

**Issue by FrankBrosU** [rafaelvcaetano/melonDS-android#123]
* **Device model:** SM-T385M
* **Android version:** Android API 28
* **melonDS version:** Beta 1.2.2

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$d: Failed to load firmware: BIOS9_MISSING
	at me.magnum.melonds.ui.emulator.h$c.a(Unknown Source:54)
	at f.a.b0.e.a.a.g(Unknown Source:10)
	at f.a.a.a(Unknown Source:14)
	at f.a.b0.e.a.e$a.run(Unknown Source:2)
	at f.a.r$a.run(Unknown Source:9)
	at f.a.b0.g.j.run(Unknown Source:13)
	at f.a.b0.g.j.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)

```

**Configuration:**  
```
FirmwareCrashContext(emulatorConfiguration=EmulatorConfiguration(useCustomBios=true, dsConfigDirectory=//, dsiConfigDirectory=//, internalDirectory=/data/user/0/me.magnum.melonds/files, fastForwardSpeedMultiplier=-1.0, useJit=false, consoleType=DS, soundEnabled=true, micSource=BLOW, firmwareConfiguration=FirmwareConfiguration(nickname=Player, message=Hello!, language=1, favouriteColour=0, birthdayMonth=1, birthdayDay=1), rendererConfiguration=RendererConfiguration(videoFiltering=NONE, threadedRendering=true)))
```

**Comment by rafaelvcaetano** [comment]
In which directory did you place your BIOS files? Looks like the app has an invalid path.

**Comment by shabbirmahmud** [comment]
Try to put your ROM files in a different directory outside the Download folder. Don't forget to update the directory in the emulator settings. I see you have the Custom BIOS option enabled. Something seems to be wrong with the directories. If you turn that option off, the ROMs should load.If it doesn't solve the problem then can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) and open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
